packages/synthetics-sdk-broken-links/src/link_utils.ts

```diff
@@ -302,6 +307,8 @@ export function setDefaultOptions(
     // per_link_options and linkorder are handled below
     if (
       !(optionName in inputOptions) ||
+      // eslint-disable-next-line  @typescript-eslint/no-explicit-any
+      (inputOptions as any)[optionName] === undefined ||
```

Comment:
wait, how does this work?

Reply:
this extra check is because 
{ origin_url: 'fake-link } is different than { origin_url: 'fake-link, link_limit: undefined }

`optionName in inputOptions` would evaluate to true in the 2nd case so I needed to add this statement